src/components/modals/SettingsModal/SettingsDialog.tsx

```
@@ -63,14 +64,44 @@
+            </span>
+        ),
         initialValue: getConfig(CONFIG_KEY.DECODER_OPTIONS).logLevelKey,
         label: "Decoder: Log level key",
         name: LOCAL_STORAGE_KEY.DECODER_OPTIONS_LOG_LEVEL_KEY,
         type: "text",
     },
     {
-        helperText: "[JSON] Key to extract the log timestamp from.",
+        helperText: (
+            <span>
+                [Structured] Key that maps to each log event's timestamp. See
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)